microcosm_pubsub/dispatcher.py

```diff
@@ -86,7 +106,9 @@ def find_handler(self, message, bound_handlers):
 
         """
         try:
-            return self.sqs_message_handler_registry.find(message.media_type, bound_handlers)
+            handler = self.sqs_message_handler_registry.find(message.media_type, bound_handlers)
+            self.opaque["handler"] = titleize(handler.__class__.__name__)
```

Comment:
Save handling information once we have it. We cannot do this during initialization of `self.opaque` since it would allow an error to be raised before the result handling (if there was no such handler)